Formula/ocrmypdf.rb

```diff
@@ -138,7 +138,7 @@ def install
                 "JPEG_ROOT = ('#{Formula["jpeg"].opt_prefix}/lib', '#{Formula["jpeg"].opt_prefix}/include')"
         s.gsub! "FREETYPE_ROOT = None",
                 "FREETYPE_ROOT = ('#{Formula["freetype"].opt_prefix}/lib', " \
-                                 "'#{Formula["freetype"].opt_prefix}/include')"
+                "'#{Formula["freetype"].opt_prefix}/include')"
```

Comment:
Terrible. Though I think we can fix this by doing `depends_on "pillow"` instead.

Reply:
Will be done in https://github.com/Homebrew/homebrew-core/pull/80458